idea/idea-frontend-api/src/org/jetbrains/kotlin/idea/frontend/api/components/KtReferenceShortener.kt

```diff
@@ -35,22 +36,40 @@ abstract class KtReferenceShortener : KtAnalysisSessionComponent() {
 }
 
 interface KtReferenceShortenerMixIn : KtAnalysisSessionMixIn {
-    fun collectPossibleReferenceShortenings(
-        file: KtFile,
-        selection: TextRange = file.textRange,
-        classShortenOption: (KtClassOrObjectSymbol) -> ShortenOption = {
+    companion object {
+        val defaultClassShortenOption: (KtClassOrObjectSymbol) -> ShortenOption = {
```

Comment:
These 2 properties can be private (unless you were planning on using it somewhere else)

Reply:
Thanks! Making them private sounds good.